newdle/client/src/actions.js

```diff
@@ -34,6 +34,9 @@ export const CHOOSE_ALL_AVAILABLE = 'Choose all slots where user is available';
 export const CHOOSE_MANUALLY = 'Manually select available slots';
 export const SET_ANSWER_BUSY_TIMES = 'Set answer busy times';
 export const PARTICIPANT_RECEIVED = 'Received participant data';
+export const ADD_ERROR = 'Error occurred';
+export const REMOVE_ERROR = 'Remove error';
+export const CLEAR_ERRORS = 'Clears all the errors';
```

Comment:
Nitpicking, but this should be in the imperative mood ("Clear all the errors"), right?

Reply:
Right, thanks!